ci/scripts/audit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
```

Comment:
It looks like we are missing execute permissions on this file:

```
chmod 755 ci/scripts/audit.sh
```
Should sort things out.